ramls/order.raml

```diff
@@ -57,7 +57,7 @@ resourceTypes:
     get:
       is: [pageable, searchable: {description: "using CQL (indexes for purchase orders)", example: "workflow_status==\"Pending\""} ]
     post:
-      description: Post a purchase order (PO) and a number of PO lines; record fund transactions corresponding to the order
+      description: Post a purchase order (PO) and a number of PO lines; record fund transactions corresponding to the order. IMPORTANT! It is required that user should have permissions orders-storage.acquisitions-unit-assignments.item.post to create an order during the call to the API. Otherwise, error 403 will be returned.
```

Comment:
please update that it's needed only if acquisition units are being assigned

Reply:
Done.